### PR TITLE
Initial draft

### PR DESCRIPTION
**Pull request by greg0ire**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,10 @@
+.PHONY: build
+
+# To be run from the host
+.env:
+	@ cp $@.dist $@
```
Comment (L-P) [inline Makefile]: -@

File: composer.json [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+        "symfony/dependency-injection": "^3.2",
+        "symfony/dotenv": "@dev",
+        "symfony/serializer": "^3.2"
+    },
+    "autoload": {
+        "psr-4": {
+            "DigitalBackstage\\OrangeJuicer\\": "src"
+        }
+    },
+    "license": "proprietary",
```
Comment (L-P) [inline composer.json]: Nope.

File: docker/docker-compose.override.yml [hunk 3]
```diff
@@ -0,0 +1,10 @@
+version: '2'
+services:
+  orange_juicer:
+    entrypoint: docker/dev-entrypoint.sh
+    volumes:
+      - ~/Vidéos:/data
```
Comment (L-P) [inline docker/docker-compose.override.yml]: Specific to your setup

File: spec/ManifestGeneratorSpec.php [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+
+class ManifestGeneratorSpec extends ObjectBehavior
+{
+    function let(FilesystemInterface $filesystem)
+    {
+        $encoder = new XmlEncoder();
+        $this->beConstructedWith(
+            $encoder,
+            $filesystem,
+            array()
```
Comment (L-P) [inline spec/ManifestGeneratorSpec.php]: PHP 5.4 is long gone, `[]`

File: src/ManifestGenerator.php [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+
+    public function generateManifest(
+        string $filePath,
+        string $title,
+        string $audioLanguage,
+        string $subtitlingLanguage = null
+    ) {
+        $metadata = [];
+        $metadata['title_1'] = $title;
+        $metadata['multi_format_set']
```
Comment (L-P) [inline src/ManifestGenerator.php]: Confusing indentation

File: src/MetadataProvider/MediaInfoProvider.php [hunk 6]
```diff
@@ -0,0 +1,76 @@
...
+    public function provideData()
+    {
+        $info = $this->mediaInfoWrapper
+            ->getInfo($this->fsRoot . $this->filePath);
+        $aspectRatio = (string) current($info->getVideos())
+            ->get('display_aspect_ratio');
+
+        list($width, $height) = explode(':', $aspectRatio);
+        /* Remove cast to int when
+         * https://github.com/mhor/php-mediainfo/issues/65 is fixed */
```
Comment (greg0ire) [inline src/MetadataProvider/MediaInfoProvider.php]: Do not merge yet, this has been fixed and released :)

File: src/TrailerDetector.php [hunk 7]
```diff
@@ -0,0 +1,29 @@
...
+
+    public function __construct(Filesystem $filesystem)
+    {
+        $this->filesystem = $filesystem;
+    }
+
+    public function getTrailerFilename(string $programFilename): string
+    {
+        return preg_replace(
+            '|MPEG\d+\.\mpg|',
```
Comment (L-P) [inline src/TrailerDetector.php]: `\m` ?

**Review by L-P — CHANGES_REQUESTED** [review]

**Review by L-P — APPROVED** [review]

**Pull request merged**